### ISSUE TITLE
It doesn't seem to be able to support content-only plugins

### ISSUE THREAD

**Issue by PrimaryFantasty** [medelfor/surreal-docs#4]

**Comment by wcobalt** [comment]
Hey @PrimaryFantasty!

We've identified the bug and working to fix it. The new version which support such ability will be available today or tomorrow

**Comment by wcobalt** [comment]
Hi @PrimaryFantasty!

We've released a new version of Surreal Docs (1.0.4), which should fix the issue you encountered. In order to generate docs for a content-only plugin `MyPlugin`, you need to do the following:

0. Install Surreal Docs `1.0.4`;
1. Navigate to the project's directory and call `surdocs init`, then `ENTER`, this will install an updated version of Surreal Docs UE plugin;
2. Open `surdocs/surreal.json` inside the project's directory and add the plugin's name by the path `filters.content.include`, such as:
```
...
    "filters": {
        "content": {
            "exclude": [],
            "include": [
                "/Game/",
                "/MyPlugin/"
            ]
        },
...
```
3. Save `surreal.json` and run `generate` as usual;

These steps are not in the documentation yet, we'll add them soon.

Be sure to try the new version out and let me know whether it fixes the issue for you

**Comment by PrimaryFantasty** [comment]
Now it can export the documentation of content-only plugins, thank you!
